src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1890,12 +1890,6 @@ public ErrorDiagnostic IndexOutOfBounds(string typeName, long tupleLength, long
                 "BCP335",
                 $"The provided value can have a length as large as {sourceMaxLength} and may be too long to assign to a target with a configured maximum length of {targetMaxLength}.");
 
-            public ErrorDiagnostic ParametersFileUnsupported() => new(
-                TextSpan,
-                "BCP336",
-                $@"Using a Bicep Parameters file requires enabling EXPERIMENTAL feature ""{nameof(ExperimentalFeaturesEnabled.ParamsFiles)}"".",
-                new Uri("https://aka.ms/bicep/config"));
-
```

Comment:
Do we ever error remove these or is way to deprecate?

Reply:
There is no way to deprecate error messages. I think it's fine to remove it.